tests/feinterface/fe_interface_view_03.cc

```diff
@@ -0,0 +1,178 @@
+// ------------------------------------------------------------------------
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2022 - 2024 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// Part of the source code is dual licensed under Apache-2.0 WITH
+// LLVM-exception OR LGPL-2.1-or-later. Detailed license information
+// governing the source code and code contributions can be found in
+// LICENSE.md and CONTRIBUTING.md at the top level directory of deal.II.
+//
+// ------------------------------------------------------------------------
+
+
+// Check FEInterfaceViews::get_*_function_values/gradients...
```

Comment:
Could you add a little bit more detail about what the accomplishes with respect to checking gradients? One line or two description :).